microsetta_interface/templates/account_details.jinja2

```diff
@@ -68,6 +68,32 @@
                     $("#state").empty();
                 }
 
+                // Special case to prevent users outside Japan from using Japanese as their language
+                $('#language').empty();
+                if (code == "JP") {
```

Comment:
Is "code" derived from language tag rather than country? Output on here is hard via phone :/

Reply:
No, it's the country that the user selects for their residence.

Reply:
Thanks!